velox/experimental/cudf/tests/FilterProjectTest.cpp

```diff
@@ -863,7 +901,13 @@ TEST_F(CudfFilterProjectTest, mixedLiteralProjection) {
   testMixedLiteralProjection(vectors);
 }
 
-TEST_F(CudfFilterProjectTest, dereference) {
+// This test checks for CudfExpression's ability to handle nested field
+// references. However, to test this, we need to disable CPU fallback, otherwise
+// the test could pass by using CPU, having not exercised the CudfExpression at
+// all. But this test relies on row_constructor to construct the nested fields,
+// which CudfExpression doesn't support. Disabling this until we have a better
+// test
+TEST_F(CudfFilterProjectTest, DISABLED_dereference) {
```

Comment:
I remember we support row_constructor in cudf CC @karthikeyann

Reply:
I can see that AST supported "nested_column" but I don't see "row_constructor" in the old commits. https://github.com/facebookincubator/velox/blob/b9bc9fd9647e3b9cd192464ca1f5a18a5896854d/velox/experimental/cudf/exec/ExpressionEvaluator.cpp#L732-L733

Reply:
Yes, this is to support row_constructor https://github.com/rapidsai/velox/pull/31/files